confiture-web-app/src/components/report/ReportCriteria.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { pluralize } from "../../utils";
+
+// FIXME: type things
+const props = defineProps<{
+  pagesData: any;
+  transverseData: any;
+  showFilters?: boolean;
+}>();
```

Comment:
```suggestion
const props = defineProps<{
  pagesData: ReportErrors[];
  transverseData: TransverseReportErrors[];
  showFilters?: boolean;
}>();
```

+ importer les types

Reply:
Ah oui, le petit oubli.